frontend/js/src/user/Listens.tsx

```diff
@@ -995,6 +1015,13 @@ export default class Listens extends React.Component<
                         this.handlePinnedRecording(pinnedListen)
                       }
                     />
+                    <MbidMappingModal
+                      recordingToMap={recordingToMapToMusicbrainz}
+                      newAlert={newAlert}
+                      onSuccessfulPin={(pinnedListen) =>
+                        this.handlePinnedRecording(pinnedListen)
+                      }
```

Comment:
just left this here for now. Is there an action that we'd want to make once a mapping is made? Should we look up this mapping in the API and add the mbid to the listen?
Note that when performing a "get listens" from the API, we'll include a manual mapping mbid in the response in the normal mbid location if it's set, so after a refresh this will correctly show